charts/px-backup/values.yaml

```diff
@@ -0,0 +1,58 @@
+namespace: px-backup
+
+customeRegistryEnabled: false
+dockerRegistryURL: docker.io
+dockerImageRepoName: portworx
+imagePullSecrets: docregistry-secret
+imagePullPolicy: Always
+
+persistentStorage:
+  enabled: false
+  storageClassName:
+
+nodeAffinityKey: pxc/enabled
+nodeAffinityValue: false
+storkRequired: false
+ingressControllerSetup: true
+centralEndpoint: None
+
+centralOIDC:
+  enabled: true
+  username: admin
+  password: admin
```

Comment:
Rename these to `defaultUsername` and `defaultPassword`

Reply:
Changed